CP11NOSM/CP11NOSM_D00002_ingest.csv

```diff
@@ -0,0 +1,39 @@
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl27/syslog/*.syslog.log,CP11NOSM-RID27-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl27/optaa1_decimated/*.optaa1.log,CP11NOSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl27/flort/*.flort.log,CP11NOSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP11NOSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl27/dosta1/*.dosta1.log,CP11NOSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/irid2shore/cpm_status*.txt,CP11NOSM-SBC11-00-CPMENG000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/syslog/*.syslog.log,CP11NOSM-SBD11-00-DCLENG000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/mopak/*.mopak.log,CP11NOSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will not be telemetered
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/hyd1/*.hyd1.log,CP11NOSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/spkir1/*.spkir1.log,CP11NOSM-SBD11-05-SPKIRB000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/metbk/*.metbk.log,CP11NOSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl11/metbk/*.metbk.log,CP11NOSM-SBD11-06-METBKA001,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl12/syslog/*.syslog.log,CP11NOSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl12/hyd2/*.hyd2.log,CP11NOSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_newsba5_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl12/pco2a/*.pco2a.log,CP11NOSM-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00002/cg_data/dcl12/wavss/*.wavss.log,CP11NOSM-SBD12-05-WAVSSA000,telemetered,Available,
```

Comment:
looks good - data on RDR.  Was the MFN PRTSZ issue figured out (in P21 slack channel)?  I see log files but doesn't look like they have data.  Should this be commented out?

Reply:
> looks good - data on RDR. Was the MFN PRTSZ issue figured out (in P21 slack channel)? I see log files but doesn't look like they have data. Should this be commented out?

The MFN PRTSZ did fail but it did produce some valid data files before failing. We will ingest those few files and annotate the failure